Website/src/rendering/src/pages/api/ordercloud/syncproducts.ts

```diff
@@ -139,6 +147,42 @@ type VariantRow = {
   size: string;
 };
 
+async function postFacets() {
+  await Promise.all([
+    ProductFacets.Save('color', {
+      ID: 'color',
+      Name: 'Color',
+      XpPath: 'Facets.color',
+      MinCount: 1,
+      xp: {
+        Options: [
+          'Red',
```

Comment:
If I understand correctly, facets and their values are not dynamically listed from the feed data. They are instead hardcoded here. Right?

Reply:
These options really only matter for the admin application. They aren't all of the options in the current data set, they are all of the options that could be set by a supplier or admin and are managed by the marketplace owner. The idea is that a marketplace owner might only want a supplier to select from a small set of colors rather than add any value here.

Reply:
Oh great then. Thanks for the explanation!